tqdm/_tqdm.py

```diff
@@ -289,9 +289,20 @@ def format_meter(n, total, elapsed, ncols=None, prefix='',
 
             # format the stats displayed to the left and right sides of the bar
             l_bar = (prefix if prefix else '') + \
-                '{0:3.0f}%|'.format(percentage)
-            r_bar = '| {0}/{1} [{2}<{3}, {4}{5}]'.format(
-                    n_fmt, total_fmt, elapsed_str, remaining_str, rate_fmt,
+                '{0:6.2f}%|'.format(percentage)
```

Comment:
I believe blaz has  a valid point.  Yes, you can use a completely custom bar, but I don't think you can overload that part of the left bar with a different sub-format. If you can, you (tqdm team) might want to add a FAQ type thing for that.